dynamiqs/mesolve/euler.py

```diff
@@ -1,10 +1,17 @@
 from torch import Tensor
 
-from ..solvers.ode.fixed_solver import FixedSolver
+from ..solvers.ode.fixed_solver import AdjointFixedSolver
 from .me_solver import MESolver
 
 
-class MEEuler(MESolver, FixedSolver):
+class MEEuler(MESolver, AdjointFixedSolver):
     def forward(self, t: float, rho: Tensor) -> Tensor:
         # rho: (b_H, b_rho, n, n) -> (b_H, b_rho, n, n)
         return rho + self.dt * self.lindbladian(t, rho)
+
+    def backward_augmented(self, t: float, rho: Tensor, phi: Tensor) -> Tensor:
```

Comment:
```suggestion
    def backward_augmented(self, t: float, rho: Tensor, phi: Tensor) -> tuple[Tensor, Tensor]:
```
Should also be fixed in `rouchon.py` and `fixed_solver.py`.